InnowisePilotWebClient/Controllers/FridgeController.cs

```diff
@@ -0,0 +1,134 @@
+﻿using InnowisePilotWebClient.Models;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+
+namespace InnowisePilotWebClient.Controllers
+{
+    public class FridgeController : Controller
+    {
+        string BaseUrl = "https://localhost:5001/";
+        // GET: FridgeController
+        public async Task<ActionResult> Index()
+        {
+            List<Fridge> FridgeInfo = new List<Fridge>();
+            var client = new HttpClient();
+            client.BaseAddress = new Uri(BaseUrl);
+            client.DefaultRequestHeaders.Clear();
+            HttpResponseMessage responseMessage = await client.GetAsync("api/Fridge");
+
+            if (responseMessage.IsSuccessStatusCode)
+            {
+                var FridgeResponse = responseMessage.Content.ReadAsStringAsync().Result;
```

Comment:
await asynchronously unwraps the result of your task, whereas just using Result would block until the task had completed.